doc/content/getting-started/api/go.md

```diff
@@ -0,0 +1,204 @@
+---
+title: "Go Code Examples"
+description: ""
+weight:
+---
+
+[Go](https://golang.org) is an open source programming language that makes it easy to build simple, reliable, and efficient software.
+
+<!-- more -->
+
+## Setup
+
+[Download](https://golang.org/dl/) and install Go. Run `go version` and make sure your version is newer than **1.15**.
+
+We will use the gRPC API of {{% tts %}}.
+
+Initialize a new Go module:
+
+```bash
+$ mkdir my_application
+$ cd my_application
+$ go mod init github.com/example/my_application
+```
+
+Add the following contents to your `go.mod` file:
+
+```go
+// go.mod
+module github.com/example/my_application
+
+go 1.15
+
+// Dependency of lorawan-stack.
+replace gopkg.in/DATA-DOG/go-sqlmock.v1 => gopkg.in/DATA-DOG/go-sqlmock.v1 v1.3.0
+
+// Dependency of lorawan-stack.
+replace gocloud.dev => gocloud.dev v0.19.0
+
+// Dependency of lorawan-stack.
+replace github.com/grpc-ecosystem/grpc-gateway => github.com/TheThingsIndustries/grpc-gateway v1.15.2-gogo
+
+require (
+    go.thethings.network/lorawan-stack/v3 v3.10.5
+    google.golang.org/grpc v1.34.0
+)
```

Comment:
Instead of encouraging people to copy and paste a dependency file, I think it's better to mention the `go` commands that lead to this. I.e. `go get` and `go mod edit -replace ...` (if the latter is really necessary)